keyboards/splitkb/kyria/rev1/config.h

```diff
@@ -85,6 +85,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #ifdef OLED_ENABLE
 #    define OLED_DISPLAY_128X64
 #    define SPLIT_OLED_ENABLE
+#    define DEFAULT_OLED_VISUALIZER
+#endif
+
+#ifdef ENCODER_ENABLE
+#    define DEFAULT_ENCODER_ACTIONS
```

Comment:
```suggestion
```

You should just need to return the correct value from `oled_task_user`, <https://docs.qmk.fm/#/ChangeLog/20211127?id=oled-task-refactor>.

Therefore all `DEFAULT_*` should be reverted.

Reply:
I feel like my point is not getting across. This is about reducing the amount of code that ends up in the image. If I return false from the user function the code from the kb function will still end up in the image uselessly.

Reply:
I understand your end goal of reducing the final firmware size. The issue is with the implementation.

My point is based on previous discussions, this sort of "bespoke" keyboard behaviour isnt one we would want in the repo, nor a paradigm to propagate and later have to unpick.

Reply:
That wasn't the case in my test (and I have LTO enabled). I understand that the changes in this PR are non-standard and controversial so I'm OK with just closing this PR and maintaining these changes in my own fork.